src/utils/padwithzeros.ts

```diff
@@ -0,0 +1,9 @@
+export function padWithZeros(num: number, maxSize: number): string {
```

Comment:
This looks to me like it could be done with `String.prototype.padStart()`?
e.g. `return num.toString().padStart(maxSize, 0);`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
Well spotted, thank you @mcadenas-bjss 👍